tests/ert/unit_tests/analysis/test_misfit_preprocessor.py

```diff
@@ -142,18 +142,29 @@ def test_that_correlated_and_independent_observations_are_grouped_separately(
 
 
 @pytest.mark.parametrize("nr_observations", [0, 1, 2])
-def test_misfit_preprocessor_single(nr_observations):
-    """We create nr_observations responses using the linear function y = ax.
-    We don`t have enough information to cluster properly, so we don`t try
-    to scale"""
-    rng = np.random.default_rng(1234)
+def test_that_perfectly_correlated_responses_are_not_scaled(nr_observations):
```

Comment:
`test_that_responses_with_correlation_one_are_not_scaled`?

Reply:
Thanks, but I prefer the original :)

Reply:
I have changed the name of the test to `test_edge_cases_with_few_observations_return_default_values`.